.github/workflows/ci.yml

```diff
@@ -21,6 +21,7 @@ env:
   SERVER_PROJECT_ID: 'github-ci-server'
   CLIENT_PROJECT_ID: 'github-ci-app-0'
   BIGQUERY_DATASET_ID: 'audit_logs.audit_abcxyz_data_access'
+  AUDIT_CLIENT_TEST_PRIVATE_KEY_PATH: './integration/testrunner/private_key.json'
```

Comment:
I think it is removed in commit ([36cc594de531d96b6f690c7f2717d0697ce5d550](https://github.com/abcxyz/lumberjack/commit/36cc594de531d96b6f690c7f2717d0697ce5d550)), so it's not here anymore.
Maybe you see this because there is some sort of bug that caused the webpage to show code from previous commit?